main.js

```diff
@@ -0,0 +1,386 @@
+let gamesPlayed = 1;
+let startTime = Date.now();
+let turnCount = 0;
+let whoseTurn = "X";
+let occupiedCells = {
+  all: [],
+  X: [],
+  O: []
+};
+let cells = [];
+let startOverButton = document.getElementById("startOver");
+startOverButton.style = "opacity: 0";
+let timer = document.getElementById("timer");
+let playerX = {
+  name: "Player X",
+  type: "Human",
+  winCount: 0
+};
+let playerO = {
+  name: "Player O",
+  type: "Human",
+  winCount: 0
+};
+let statusMessage = document.getElementById("status");
+statusMessage.textContent = XOToPlayerName(whoseTurn) + "'s turn!";
+let changeNamesButton = document.getElementById("changeNames");
+let currentState = "gameStarted";
+let winningTriple = [];
+let playerXNameInput = document.getElementById("playerXName");
+let playerONameInput = document.getElementById("playerOName");
+let playerXWinCount = document.getElementById("playerXwins");
+let playerOWinCount = document.getElementById("playerOwins");
+
+let states = {
```

Comment:
This is an interesting way to make it so you can't start a new game while you're already playing.  It may have been simpler to just disable the start button though.